src/content/docs/browser/new-relic-browser/configuration/obfuscate-browser-agent-data.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Obfuscate Browser Agent Data
+tags:
+  - Browser monitoring
+  - Obfuscation
+metaDescription: "The agent can be configured to obfuscate the data it sends."
+---
+
+<Callout variant="important">
+This feature is currently available for those using the copy/paste or NPM browser installation methods. There is currently no UI or NerdGraph configuration options available. We are continuing to work on improving access to these and other configuration options.
+</Callout>
+
+While New Relic's recommendation is to abstain from utilizing sensitive information in the public structure of your application, we also understand that it is not always feasible to remove all traces of said information. The browser agent can be configured to selectively obfuscate data in every payload it sends.  This can be useful if your application uses sentitive data in places that the agent captures, such as navigation paths, error messages, and more.
+
+## How it works [#how-it-works]
+
+As of browser agent [version 1216](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, obfuscation rules can be applied to outgoing harvest payloads. 
+
+To set up these rules, you'll need to configure the following browser agent properties: 
+* `init.obfuscate` contains an array of selectors and replacements which will be utilized to modify each harvest before sending.
+  * If you’re using the copy/paste installation method, you’ll need to manually edit your JavaScript snippet and set the `obfuscate` array to contain your obfuscation conditions.
+
+### Recommendations
+
+When configuring these properties, we recommend the following:
+
+* Use intentional regex patterns to obfuscate only what needs obfuscation.
+  - Needlessly obfuscating can have side-effects such as less granularity when grouping data and less ability to digest what the agent captured
+* Replace your sensitive data with neutral and generic terms that also indicate what data has been redacted.
+  - Example: `/account-id/g` --> `ACCOUNT_ID`
+
+### Copy/paste installation [#copy-paste]
+
+If you're using the copy/paste installation method, add the following configuration to your browser JavaScript configurations before the agent loader:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>...,
+    obfuscate: [
+      {
+        regex: <RegExp | string>
+        replacement: <string>
+      },
+      ...<other obfuscation rules>...
+    ]
+}
+```
+
+
+### NPM installation [#npm]
```

Comment:
At the top of the doc, we're saying that the copy/paste method is the only way to make configuration changes, but down here in the doc, it looks like you can make changes using NPM. Can you clarify that at the top of the doc?

Reply:
It already calls out NPM, or is there a spot I'm not seeing?

![image](https://github.com/newrelic/docs-website/assets/24296585/027a1e26-84fb-49c8-ad1a-dc046855125c)